wordpress-mobile/WordPress-Android
# Use binary configure

This PR uses the gradle `configure` plugin instead of bundler, which should remove the need for a ruby toolchain for day-to-day development of WPAndroid.

**To Test:**

- Ensure all CI things pass
- Check out the repo from scratch, then switch to this branch. Run ./gradlew applyConfiguration. Note that it fails with the error "Build configuration file gradle.properties doesn't exist, follow README instructions". Copy gradle.properties-example to gradle.properties, then run ./gradlew applyConfiguration. This should replace the default values with the proper ones.

PR submission checklist:
